### PR TITLE
Fixed errors in proxys storyboard generation

### PR DESCRIPTION
**Pull request by LedaLima**
Incident progression file was not being populated correctly
Timeline chart was being overwritten on every new storyboard creation

### CODE REVIEW

**Comment by vgonzale78** [comment]
+1

File: spot-oa/api/resources/proxy.py [hunk 1]
```diff
@@ -179,33 +179,28 @@ def get_scored_requests(date):
...
-               'resconttype':row['responseContentType']})
+           referers[row['referer']]+=1
+           requests += [{'clientip':row['clientIp'], 'referer':row['referer'],'reqmethod':row['requestMethod'], 'resconttype':row['responseContentType']}]
+
         else:
             #Destination URI's refered by the threat
             refered[row['uri']]+=1
 
-    create_incident_progression(uri,requests,referers,date)
+    create_incident_progression(uri,requests,refered,date)
```
Comment (rabarona) [inline spot-oa/api/resources/proxy.py]: How does replacing _referers_ with _refered_ fixes this issue?

File: spot-oa/api/resources/proxy.py [hunk 2]
```diff
@@ -217,23 +212,30 @@ def create_storyboard(uri,date,title,text,expanded_search,top_results):
...
 """
 def create_timeline(anchor,clientips,date,top_results):
-
     response = ""
     susp_ips = []
 
-
     if clientips:
-	srtlist = sorted(list(clientips.items()), key=lambda x: x[1], reverse=True)
+	    srtlist = sorted(list(clientips.items()), key=lambda x: x[1], reverse=True)
```
Comment (rabarona) [inline spot-oa/api/resources/proxy.py]: Is this indentation correct? If yes, is line 220 (for val in strlist[:top_results]: indentation correct?

Comment (LedaLima) [inline spot-oa/api/resources/proxy.py]: these indentations are not correct, this  is being fixed right now

File: spot-oa/api/resources/proxy.py (hunk 1 above)
Comment (LedaLima) [inline spot-oa/api/resources/proxy.py]: @vgonzale78 can answer this question

**Comment by raypanduro** [comment]
+1

**Pull request merged**